src/shared-content/installation/migrate-from-windows-to-linux-container.include.md

```diff
@@ -96,11 +96,16 @@ Our recommendation is to keep that risk to a minimum.
 
 ### Copy Files
 
-Octopus Deploy stores all the BLOB data (deployment logs, runbook logs, packages, artifacts, etc.) on a file share.  If you are moving from a single server, be it hosting Octopus in a Windows Container or directly on a Windows VM, you will need to copy files to your new storage provider.  Once your shared storage provider has been created, you'll want to copy files over from these folders:
+Octopus Deploy stores all the BLOB data (deployment logs, runbook logs, packages, artifacts, event exports etc.) on a file share.  If you are moving from a single server, be it hosting Octopus in a Windows Container or directly on a Windows VM, you will need to copy files to your new storage provider.  Once your shared storage provider has been created, you'll want to copy files over from these folders:
 
 - TaskLogs
 - Artifacts
 - Packages
+- EventExports
+
+:::div{.hint}
```

Comment:
I don't think this hint block is needed here

Reply:
Deleted